tests/python/unittest/test_tvm_op.py

```diff
@@ -24,14 +24,13 @@
 
 @with_seed()
 def test_tvm_broadcast_add():
-    if _features.is_enabled("TVM_OP"):
```

Comment:
Lets keep that if possible

Reply:
I will, thanks for reminding.  could you guide me where to pip install package in docker? ci failed with "ImportError: No module named 'decorator'" during compiling